app/lib/foreman_remote_execution/renderer/scope/input.rb

```diff
@@ -23,6 +42,14 @@ def cached(key, &block)
           Rails.cache.fetch(cache_key, &block)
         end
 
+        apipie :method, 'Render template by given name' do
+          required :template_name, String, desc: 'name of the template to render'
+          optional :input_values, Hash, desc: 'key:value list of input values for the template'
+          optional :options, Hash, desc: 'TODO: not sure'
```

Comment:
Not sure about usage of this one...

Reply:
If I'm reading this right so far the only expected key in there is `:with_foreign_input_set` which is expected to be a boolean and controls whether a foreign input set should be considered when rendering the template.

Reply:
Thought so too, thanks. Updated.